enterprise/internal/campaigns/resolvers/changeset_event.go

```diff
@@ -0,0 +1,35 @@
+package resolvers
+
+import (
+	"github.com/graph-gophers/graphql-go"
+	"github.com/graph-gophers/graphql-go/relay"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/graphqlbackend"
+	ee "github.com/sourcegraph/sourcegraph/enterprise/internal/campaigns"
+	"github.com/sourcegraph/sourcegraph/internal/campaigns"
+	"github.com/sourcegraph/sourcegraph/internal/httpcli"
+)
+
+type changesetEventResolver struct {
+	store       *ee.Store
+	httpFactory *httpcli.Factory
+	changeset   graphqlbackend.ExternalChangesetResolver
```

Comment:
```suggestion
	changesetResolver   *changesetResolver
```

I think it's better to only use the `graphqlbackend` interfaces at the boundary of our package, but not internally.

The explicit name is also better, IMHO.